flink-runtime/src/test/java/org/apache/flink/runtime/state/TaskExecutorLocalStateStoresManagerTest.java

```diff
@@ -46,9 +48,15 @@ public void testCreationFromConfig() throws Exception {
 
 		final Configuration config = new Configuration();
 
-		final String rootDirString = "localStateRoot1,localStateRoot2,localStateRoot3";
+		String tmpDir = System.getProperty("java.io.tmpdir") + File.separator;
```

Comment:
I think it's better to use a `TemporaryFolder` if there are any files created to ensure a proper clean up at the end of the test.

Reply:
👍